GPU/GLES/FragmentShaderGenerator.cpp

```diff
@@ -15,12 +15,12 @@
 // Official git repository and contact information can be found at
 // https://github.com/hrydgard/ppsspp and http://www.ppsspp.org/.
 
-#if !defined(USING_GLES2)
 // SDL 1.2 on Apple does not have support for OpenGL 3 and hence needs
 // special treatment in the shader generator.
 #if defined(__APPLE__)
-#define FORCE_OPENGL_2_0
-#endif
+const bool forceOpenGL2_0 = true;
```

Comment:
I can't help but feel that this should live somewhere else, maybe in gl_extensions? Having local bools in both the fragment and vertex gens feels wrong.